### PR TITLE
Ensuring logits scores are contiguous resolved the bug on MPS

### PR DESCRIPTION
**Pull request by LeoN0425**
在Mac的MPS框架上运行代码时，发现性能表现不佳。这是由于CustomRepetitionPenaltyLogitsProcessorRepeat处理在MPS上运行时，导致score变量出现数值错误的异常。将scores变量进行连续内存排布处理，即使用scores.contiguous()方法，可以解决这个问题。

### CODE REVIEW

**Review by fumiama — APPROVED** [review]
我咋记得我合过这个PR了😂

**Comment by fumiama** [comment]
哦，你弄错分支了
